### PR TITLE
SCB-2097 Synchronize some bug fix codes

### PR DESCRIPTION
**Pull request by develpoerX**

### CODE REVIEW

File: service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceInstanceStatusSyncTask.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
+package org.apache.servicecomb.serviceregistry.task;
```
Comment (liubao68) [inline service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceInstanceStatusSyncTask.java:1]: add license to to file

**Comment by liubao68** [comment]
Please create more specific issues for this PR. 

e.g

1. upgrade xxx from x to y. e.g.  https://issues.apache.org/jira/browse/SCB-2018
2. fix bug: xxxx

You can still use one PR to fix them all. These issues will used in Release Notes.

File: service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceInstanceStatusSyncTask.java (hunk 1 above)
Comment (develpoerX) [reply service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceInstanceStatusSyncTask.java:1]: Ok, Thanks for you suggestion.

**Comment by develpoerX** [comment]
> Please create more specific issues for this PR.
> 
> e.g
> 
> 1. upgrade xxx from x to y. e.g.  https://issues.apache.org/jira/browse/SCB-2018
> 2. fix bug: xxxx
> 
> You can still use one PR to fix them all. These issues will used in Release Notes.

**Comment by develpoerX** [comment]
I have modified it by putting the sub-problem description and linking in the description

**Pull request closed without merging**